src/arena.py

```diff
@@ -1 +1,108 @@
-WORKS = True
+import json
+import re
+import random
+from typing import List
+
+
+class Damage:
+    __x: int
```

Comment:
Please name your variables in a more descriptive way :)

Reply:
ok